out/production/resources/application.properties

```diff
@@ -1,12 +1,11 @@
-spring.data.neo4j.uri=bolt://35.225.138.96:7687
 
+spring.data.neo4j.uri=bolt://35.225.138.96:7687
 spring.data.neo4j.username=neo4j
-spring.data.neo4j.password=pass
+spring.data.neo4j.password=password
+
 google.cloud.bucket=travelbook
 google.cloud.credentials=src/main/resources/Bachelor-bc0312b7b74b.json
 
-jwt.token.secret=jwtappdemo
-jwt.token.expired=10000000
-
-
+token.secret=appdemo
+token.expired=3600000
```

Comment:
There is try to rewrite current time of JWT token.

Reply:
This change throw exception